src/components/Header/Header.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
-import { Link } from "react-router-dom";
+import { Link } from 'react-router-dom';
+import classNames from "classnames";
```

Comment:
This import is required to make lines 14 and 31 work correctly. Confusingly, `classNames` is a library, while `className` (singular) is a property built into React/JSX